packages/__docs__/src/Document/index.tsx

```diff
@@ -199,6 +200,28 @@ class Document extends Component<DocumentProps, DocumentState> {
     )
   }
 
+  renderThemeLink() {
+    const { srcUrl, srcPath, legacyGitBranch } = this.props.doc
+    if (!srcUrl) return
+
+    const themeUrl = srcUrl.replace('index.tsx', 'theme.ts')
+
+    const themePath = srcPath.replace('index.tsx', 'theme.ts')
+
+    let legacySrcUrl
+
+    if (legacyGitBranch) {
+      legacySrcUrl = themeUrl.replace('/master/', `/${legacyGitBranch}/`)
+    }
+
+    return (
+      <View as="div" margin="0 0 x-large 0">
+        You can check these variables at{' '}
```

Comment:
Text suggestion: `See which global theme variables are mapped to the component here: [link]`